R/init.R

```diff
@@ -224,7 +224,7 @@ init <- function(data,
     }
 
     is_modules_ok <- check_modules_datanames(modules, .teal_data_datanames(data))
-    if (!isTRUE(is_modules_ok)) {
+    if (!isTRUE(is_modules_ok) && length(unlist(extract_transformers(modules))) == 0) {
       lapply(is_modules_ok$string, logger::log_warn)
```

Comment:
If you change it to warning you'd be able to `testthat::expect_warning`
```suggestion
lapply(is_modules_ok$string, warning, call. = FALSE)
```